src/test/common/page/terms-and-conditions.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const webdriver = require('selenium-webdriver')
+const SubmittablePage = require('./submittable-page')
+const TERMS_AND_CONDITIONS_TITLE = 'GOV.UK - Read and agree to our terms and conditions'
+
+/**
+ * Page object for the page where the customer can check their details before submitting.
+ */
+class TermsAndConditions extends SubmittablePage {
+  getPath () {
+    return '/terms-and-conditions'
+  }
+
+  getPageName () {
+    return 'terms and conditions'
+  }
+
+  async waitForPageLoad () {
+    return super.waitForPageWithTitle(TERMS_AND_CONDITIONS_TITLE)
+  }
+
+  async selectCheckbox () {
+    const checkbox = await this.getCheckbox()
```

Comment:
As there's a few steps in here, I'd wrap this in a try/catch block to help out debugging should it fail at some point.

Reply:
done